### PR TITLE
Restructure OData Query Options document

### PR DESCRIPTION
**Pull request by sbont**
NB: this changes the URL of the document.

### CODE REVIEW

File: content/en/docs/appstore/modules/sap/sap-odata-connector.md [hunk 1]
```diff
@@ -627,7 +627,7 @@ SELECT * FROM SalesOrderSet WHERE Category='Notepads'
 
 The `$inlinecount=allpages` clause asks OData to return a count of the number of objects returned in the list. This will be stored in SAPODataConnector.ResultInfo.totalCount.
 
-You can find more information about OData queries in [OData Query Options](/refguide/odata-query-options/).
+You can find more information about OData queries in [Supported OData Operations](/refguide/supported-odata-operations/).
```
Comment (joostverhoog) [inline content/en/docs/appstore/modules/sap/sap-odata-connector.md]: The change of the title now suggest that `/refguide/odata-query-options/` describes the OData operations that the SAP OData connector supports, but that's not the case. It's built without the 'consumed OData service' feature.

I suggest we undo the changes (2x) in the title and contact the owners of the SAP OData connector so they can make changes if necessary.

File: content/en/docs/refguide/modeling/integration/odata-services/published-odata-services/supported-odata-operations.md [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 
 {{% alert color="info" %}}
-Mendix currently supports only the options described here.
+Mendix currently supports only the operations described here.
 {{% /alert %}}
 
+A published OData service is automatically documented. Studio Pro allows you to [export the OpenAPI specification](/refguide/published-odata-services/#openapi), which is a machine-readable file that is supported by most API tools, like Swagger UI and Postman.
+When running your app, you can [use Swagger UI](/refguide/build-odata-apis/#22-openapi-3-contract-and-test-page) to read this documentation and test your service. 
+
+Below follows a more descriptive explanation of the operations that your OData service will support.
```
Comment (joostverhoog) [inline content/en/docs/refguide/modeling/integration/odata-services/published-odata-services/supported-odata-operations.md]: that _a_ publised OData service will support

File: content/en/docs/appstore/modules/sap/sap-odata-connector.md (hunk 1 above)
Comment (sbont) [reply content/en/docs/appstore/modules/sap/sap-odata-connector.md]: Good point. By "changes (2x)", which changes exactly do you mean? The title of the document and the reference here? Should we change it to "OData operations" instead?

Comment (joostverhoog) [reply content/en/docs/appstore/modules/sap/sap-odata-connector.md]: I mean both changes in this document (sap-odata-connector.md).

Comment (sbont) [reply content/en/docs/appstore/modules/sap/sap-odata-connector.md]: Done.

**Comment by quinntracy** [comment]
Hello @joostverhoog @sbont is this PR ready for review or is it still being worked on?

**Comment by sbont** [comment]
@quinntracy it's ready now! :)

**Review by quinntracy — APPROVED** [review]
This PR has been approved and can be merged.
